core-httpclient-impl/build.gradle

```diff
@@ -3,6 +3,7 @@ targetCompatibility = 1.6
 
 dependencies {
     compile project(':core-api')
+    testCompile project(':core-api').sourceSets.test.output
```

Comment:
necessary to pull in `ExhaustiveTest` category (even though it's not explicitly used by the core-httpclient-impl module test